cache/cloudfront_stage.tf

```diff
@@ -24,4 +24,6 @@ module "stage_wc_org_cloudfront_distribution" {
   cache_policies    = module.cloudfront_policies.cache_policies
   request_policies  = module.cloudfront_policies.request_policies
   response_policies = module.cloudfront_policies.response_policies
+
+  waf_ip_allowlist = [local.ci_vpc_nat_elastic_ip]
```

Comment:
So we don't forget why we put this in place, how about moving this to a local and explaining we're doing this so WAF doesn't throttle the e2e tests in the front-end repo?